ideascube/utils.py

```diff
@@ -1,8 +1,27 @@
+from io import TextIOWrapper
 import sys
 
 from django.conf import locale
 
 
+class AutomagicTextIOWrapper(TextIOWrapper):
```

Comment:
That comment would fit where we use the `AutomagicTextIOWrapper` (by the way, I dislike this name and would welcome a suggestion for a better one).

But then we'd write that comment everywhere we use it?